priv/www/api/index.html

```diff
@@ -403,6 +443,10 @@ <h2>Reference</h2>
         <td>X</td>
         <td></td>
         <td class="path">/api/exchanges/<i>vhost</i>/<i>name</i></td>
+        <td>management</td>
+        <td>User added to vhost<br/>
```

Comment:
A user cannot be "added to a vhost". A user can have no or some permissions in a vhost.